Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/GenericCommandExecutionAndEventPublication.php

```diff
@@ -224,6 +224,8 @@ public function eventNumberIs(int $eventNumber, string $eventType, TableNode $pa
                 $actual = DimensionSpacePointSet::fromArray($actualValue);
                 Assert::assertTrue($expected->equals($actual), 'Actual Dimension Space Point set "' . json_encode($actualValue) . '" does not match expected Dimension Space Point set "' . $assertionTableRow['Expected'] . '"');
             } else {
+//                die();
```

Comment:
weird, I am not even sure why I woudl want to do something here^^ Removing

Reply:
?